src/main/java/org/codehaus/mojo/gwt/shell/CompileMojo.java

```diff
@@ -427,8 +427,10 @@ public void doExecute( )
         Set<Artifact> artifacts = getProject().getArtifacts();
         for (Artifact artifact : artifacts) {
             // Store the vaadin version
-            if ("vaadin-server".equals(artifact.getArtifactId())) {
+            if ("vaadin-server".equals(artifact.getArtifactId()) ||
+                    "vaadin-server-mpr-jakarta".equals(artifact.getArtifactId())) {
                 vaadinVersion = artifact.getVersion();
+                break;
```

Comment:
If the version is null, it could throw a better exception that would include the artifact list